code/modules/mob/living/carbon/xenomorph/Abilities.dm

```diff
@@ -360,7 +360,7 @@
 
 /datum/action/xeno_action/emit_pheromones/can_use_action()
 	var/mob/living/carbon/Xenomorph/X = owner
-	if(X && !X.is_mob_incapacitated() && !X.lying && !X.buckled && (!X.current_aura || X.plasma_stored >= plasma_cost) && !X.stagger)
+	if(X && !X.is_mob_incapacitated() && !X.lying && !X.buckled && (!X.current_aura || X.plasma_stored >= plasma_cost) && !X.stagger && !X.fire_stacks) //Can't emit pheromones while on fire!
```

Comment:
```suggestion
	if(X && !X.is_mob_incapacitated() && !X.lying && !X.buckled && (!X.current_aura || X.plasma_stored >= plasma_cost) && !X.stagger && !X.on_fire) //Can't emit pheromones while on fire!
```
`fire_stacks` can be positive or negative, if the mob is drenched on water or fuel.

Reply:
Fuck I keep interchanging the two